investpy/__init__.py

```diff
@@ -578,25 +616,34 @@ def get_historical_data(equity, country, from_date, to_date, as_json=False, orde
                     if interval_counter < interval_limit:
```

Comment:
All the changes done not related with `cross_currencies` should not be included since the previous code was properly formatted and the published changes are just modifications on how the data is tabulated or presented to the programmer. So **just the additions related to cross currencies data retrieval will be taken into consideration**, unless changes in any other function improves it.